### DISCUSSION TITLE
Need advice for improving optimization quality

### DISCUSSION THREAD

**Discussion by liangsihan** [simnibs/simnibs#366] [8. Simulation workflows]
Hi all,

I wrote an optimization script to find the best electrode montage for stimulating the medial prefrontal cortex(mPFC). However, it seemed that the dorsal lateral PFC was also stimulated in gmsh visualization, which is something I want to avoid. I tried to manipulate several parameters, such as target intensity, adding multiple regions along the midline/longitudinal fissure, and adding regions to aviod. But I still couldn't get the ideal result. As such, could you give me some advice to improve the focality so that the electrical field is strongest along the midline and attentuates towards the dorsal lateral part? I attached several screenshots to show my current results. Thank you
<img width="675" alt="Screenshot 2024-06-12 at 2 06 42 AM" src="https://github.com/simnibs/simnibs/assets/86599135/e1327608-3288-462f-8c97-cdc217061485">

<img width="586" alt="Screenshot 2024-06-12 at 5 35 14 AM" src="https://github.com/simnibs/simnibs/assets/86599135/12634039-e4e4-480c-b83c-1085c97a5046">

**Comment by bjornsig** [comment]
Can you try to decrease the target intensity? Because then there is current left over to cancel the electric field in other areas. If you have more questions, you can post your code so we can help you better. It also depends on how dense your grid is.
